recipes/list_items/contact_info/contact_info.vue

```diff
@@ -131,6 +132,16 @@ export default {
       default: '',
     },
 
+    /**
+     * The size of the avatar
+     * @values xs, sm, md, lg, xl
+     */
+    avatarSize: {
+      type: String,
+      default: 'lg',
+      validator: (size) => Object.keys(AVATAR_SIZE_MODIFIERS).includes(size),
+    },
```

Comment:
DtAvatar already has validator, so its throwing double console for the same error

I dont know if it is necessary to have this validator also here in the recipe...

![image](https://user-images.githubusercontent.com/89984179/225041549-140d34d0-623b-499a-978a-cafff7bbc96f.png)

Reply:
The validator is removed, please review.